### PR TITLE
Added a python script explaining how to integration test mlflow with UC OSS

### PR DESCRIPTION
**Pull request by kriscon-db**
**PR Checklist**

- [ ] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Added a script that acts as an integration test.

### CODE REVIEW

File: tests/mlflow_integration_test.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+model_version = 1
+model_uri = f"models:/{model_name}/{model_version}"
+rm_desc = "UC-OSS/MLflow Iris model"
+mv_desc = "Version 1 of the UC-OSS/MLflow Iris model"
+
+# Load the model and do some batch inference.
+X, y = datasets.load_iris(return_X_y=True, as_frame=True)
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
+loaded_model = mlflow.pyfunc.load_model(f"models:/{model_name}/{model_version}")
```
Comment (ravivj-db) [inline tests/mlflow_integration_test.py:81]: nit : you have already assigned this string to `model_uri`

Comment (ravivj-db) [reply tests/mlflow_integration_test.py:81]: also can we write a comment mentioning how this interacts with UC ?

File: tests/mlflow_integration_test.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+predictions = loaded_model.predict(X_test)
+iris_feature_names = datasets.load_iris().feature_names
```
Comment (ravivj-db) [inline tests/mlflow_integration_test.py:83]: any assertions on these ?

File: tests/mlflow_integration_test.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+print(path)
+mlflow.artifacts.list_artifacts(f"models:/{model_name}/{model_version}")
+
+# COMMAND ----------
+
+mlflow.artifacts.download_artifacts(
+  artifact_uri=f"models:/{model_name}/{model_version}",
+  dst_path=path,
+)
+os.system(f"cat /tmp/models/{model_name}/{model_version}/requirements.txt")
```
Comment (ravivj-db) [inline tests/mlflow_integration_test.py:102]: any assertions on this ?

Comment (kriscon-db) [reply tests/mlflow_integration_test.py]: fixed.

File: tests/mlflow_integration_test.py (hunk 2 above)
Comment (kriscon-db) [reply tests/mlflow_integration_test.py]: fixed.

File: tests/mlflow_integration_test.py [hunk 4]
```diff
@@ -75,31 +81,40 @@ def build_model():
...
 # COMMAND ----------
 
-mlflow.artifacts.download_artifacts(
-  artifact_uri=f"models:/{model_name}/{model_version}",
-  dst_path=path,
-)
-os.system(f"cat /tmp/models/{model_name}/{model_version}/requirements.txt")
+try:
+    mlflow.artifacts.download_artifacts(
+    artifact_uri=f"models:/{model_name}/{model_version}",
```
Comment (ravivj-db) [inline tests/mlflow_integration_test.py:124]: nit : this variable is also `model_uri`

**Review by ravivj-db — APPROVED** [review]
LGTM

**Pull request merged**